src/mavis/summary/summary.py

```diff
@@ -190,45 +192,120 @@ def group_by_distance(calls, distances):
     return grouped_calls, removed_calls
 
 
-def annotate_dgv(bpps, dgv_regions_by_reference_name, distance=0):
+def annotate_with_dgv_bpp(
+    bpps: List[BreakpointPair],
+    dgv_regions_list: List[BreakpointPair],
+    input_cluster_radius,
+):
     """
-    given a list of bpps and a dgv reference, annotate the events that are within the set distance of both breakpoints
+    Given a list of breakpoint pairs (bpps) and bpps from another reference set), annotate the events that are within the set distance of both breakpoints
 
     Args:
         bpps (list) : the list of BreakpointPair objects
-        dgv_regions_by_reference_name (dict) : the dgv reference regions file loaded by load_masking_regions
-        distance (int) : the minimum distance required to match a dgv event with a breakpoint
+        dgv_regions_list (list) : list of BreakpointPair objects specified by the MAVIS input file
+        cluster_radius (int) : Distance used in matching input SVs to reference SVs through clusterind, defined by summary.cluster_radius in the configuration file
+    """
+    bpps_copy = bpps.copy()
+    for variant in bpps:
+        variant.data[COLUMNS.dgv] = False
+        variant.data[COLUMNS.known_sv_count] = 0
+    for variant in dgv_regions_list:
+        variant.data[COLUMNS.dgv] = True
+        variant.data[COLUMNS.known_sv_count] = 0
+    bpps_copy.extend(dgv_regions_list)
+
+    clusters = merge_breakpoint_pairs(
+        bpps_copy, cluster_radius=input_cluster_radius
+    )  # all breakpoint pairs (bpps + dgv)
+
+    flag_col = '_is_variant'
+    for variant in bpps:
+        variant.data[flag_col] = True
+        variant.data[COLUMNS.dgv] = ''
+        variant.data[COLUMNS.known_sv_count] = 0
+
+    for clustered_bpp_key, clustered_bpp_val in clusters.items():
+        dgv_tracking_ids = [
+            bpp.tracking_id for bpp in clustered_bpp_val if bpp.data.get(flag_col) != True
+        ]
+        dgv_field = ';'.join(sorted(dgv_tracking_ids))
+        variant_bpp = [bpp for bpp in clustered_bpp_val if bpp.data.get(flag_col) == True]
+        for variant in variant_bpp:
+            variant.data[COLUMNS.dgv] = dgv_field
+            variant.data[COLUMNS.known_sv_count] = len(dgv_tracking_ids)
+
+    for bpp in bpps:
+        if flag_col in bpp.data:
+            del bpp.data[flag_col]
+
+
+def annotate_with_dgv_bed(
+    bpps: List[BreakpointPair],
+    dgv_regions_by_reference_name: Dict[Tuple[str], List[BreakpointPair]],
+    input_cluster_radius,
+):
+    """
+    Given a list of breakpoint pairs (bpps) and bpps from another reference set), annotate the events that are within the set distance of both breakpoints
+
+    Args:
+        bpps (list) : the list of BreakpointPair objects
```

Comment:
these also need updating to match the names

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).